e2e-tests/package.json

```diff
@@ -40,5 +44,16 @@
     "del": "^8.0.0",
     "minimist": "^1.2.8",
     "mocha-junit-reporter": "^2.2.1"
+  },
+  "nyc": {
```

Comment:
I wonder if it isn't better to put the nyc config in a separate nyc configuration file for better visibility

Reply:
extracted